src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
random shouldn't be a local variable, but a class field, same for color supplier
make 100 a class constant, use CONSTANT_CASE for naming, don't forget appropriate access modifier for all of these